src/main/java/EulerOne.java

```diff
@@ -0,0 +1,14 @@
+import java.util.stream.IntStream;
+
+public class EulerOne {
+
+    private EulerOne(){}
+
+    final static private int MIN_RANGE = 1;
```

Comment:
To powinno być nad konstruktorem. Generalnie stałe są zawsze jako pierwsze wylistowane w klasie.

Reply:
A no i to `private` raczej na początek. Wszyscy się RACZEJ trzymają konwencji `private static final ...`